main.py

```diff
@@ -26,6 +29,8 @@ def setup(self):
         log.debug("flow-manager starting")
         self._flow_mods_sent = OrderedDict()
         self._flow_mods_sent_max_size = FLOWS_DICT_MAX_SIZE
+        self.cookie_exception_range = CONSISTENCY_COOKIE_EXCEPTION_RANGE
+        self.table_id_exception_range = CONSISTENCY_TABLE_ID_EXCEPTION_RANGE
```

Comment:
Suggestion: add validation to check if the configuration value is a list of integers.

Reply:
At commit https://github.com/kytos/flow_manager/pull/112/commits/1477b0eec48fdf253d25128ebf9246e5eea2aedf, a validation was added.